crates/subspace-node/src/commands/wipe.rs

```diff
@@ -12,8 +11,6 @@ pub struct WipeOptions {
 }
 
 pub fn wipe(WipeOptions { base_path }: WipeOptions) -> Result<(), io::Error> {
-    init_logger();
```

Comment:
I think wipe needs to keep init_logger(), because it doesn’t call the config function that initialises the logger. (But I didn’t check, please let me know if I’m wrong!)